src/renderer/redux/modules/services/constants.js

```diff
@@ -1,13 +1,17 @@
 export const API_BASE = process.env.API_BASE || 'https://mofb-api.appspot.com';
-export const API_PREFIX = '/api/v2';
+export const API_PREFIX = '/api';
 export const API_URL = API_BASE + API_PREFIX;
 
-export const FOOD_CAT_ID = '11';
```

Comment:
These const's are used elsewhere in the app - did you check for other occurences? You should be able to just change the value here to `563`. And then you can change the label down below accordingly. It is easier to have the category ID const's available for the buttons in the navigation bar.

Reply:
I did check for them, and they're not currently used. It's possible that that's related to task #65, I'm looking into it now